ui/pages/onboarding-flow/creation-successful/creation-successful.tsx

```diff
@@ -209,13 +221,32 @@ export default function CreationSuccessful() {
             currentWindow: true,
           });
           if (tabs && tabs.length > 0) {
-            await browserWithSidePanel.sidePanel.open({
-              windowId: tabs[0].windowId,
-            });
+            // We deliberately skip the opening of the side panel
+            // if a user is coming from a deep link
+            if (shouldOpenSidePanel) {
+              await browserWithSidePanel.sidePanel.open({
+                windowId: tabs[0].windowId,
+              });
+              setIsSidePanelOpen(true);
+            }
             await dispatch(setUseSidePanelAsDefault(true));
             // Use the sidepanel-specific action - no navigation needed, sidepanel is already open
             await dispatch(setCompletedOnboardingWithSidepanel());
-            setIsSidePanelOpen(true);
+
+            if (deferredDeepLinkResult) {
+              if (
+                deferredDeepLinkResult.type ===
+                DeferredDeepLinkRouteType.Redirect
+              ) {
+                window.open(deferredDeepLinkResult.url, '_self');
```

Comment:
`window.open` with the target set to `_self` seems strange here. Can't tell you why, but it just seems wrong. haha. `window.location.assign(url)` or `window.location.href = url` might be better?